util/settings/settings.go

```diff
@@ -94,6 +94,15 @@ type OIDCConfig struct {
 	RequestedIDTokenClaims map[string]*oidc.Claim `json:"requestedIDTokenClaims,omitempty"`
 }
 
+type HelmRepoCredentials struct {
```

Comment:
// deprecated?

Reply:
done

Reply:
I think you might be able to extend `Repository` in Go